src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java

```diff
@@ -148,11 +148,15 @@ public AcLoadFlowResult run() {
 
         LOGGER.debug(Markers.PERFORMANCE_MARKER, "Ac loadflow ran in {} ms", stopwatch.elapsed(TimeUnit.MILLISECONDS));
 
-        AcLoadFlowResult result = new AcLoadFlowResult(networks, outerLoopIterations, nrIterations, runningContext.lastNrResult.getStatus(),
+        AcLoadFlowResult result = new AcLoadFlowResult(network, outerLoopIterations, nrIterations, runningContext.lastNrResult.getStatus(),
                 runningContext.lastNrResult.getSlackBusActivePowerMismatch());
 
         LOGGER.info("Ac loadflow complete (result={})", result);
```

Comment:
Can you please add the number of the network in the log when AC loadflow complete ?

Reply:
Ok

Reply:
Done